### DISCUSSION TITLE
Welcome to A189A Discussions!

### DISCUSSION THREAD

**Discussion by dpor16** [dpor16/A189A#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dpor16** [comment]
hello guys.

**Comment by KhairatTijani** [comment]
hello Oladipo. I posted our research questions we all worked on earlier this week. Thank you once again for presenting the question on Wednesday.  According to our task delegation the next step is data preprocessing and this was delegated to Azeez.
Hello Azeez, please kindly update us on how it is going at every stage so we can work within our deadline for next week. Thank you

**Reply by ao23aeq** [reply]
Thank you for presenting Oladipo. I'm working on it and i will send you the update before the end of tomorrow

**Comment by dpor16** [comment]
Hi guys, pls lets all go thru the task deligation and pls work on our scheduled task ahead of time

**Reply by ao23aeq** [reply]
Hi Dips. I am working on the data preprocessing and I need to clean the data and change the datatype of our variables, currently running into some errors, if I can’t figure it out by tonight, i’d create a zoom meeting for us to figure it out together! Thanks.

**Comment by feekieboy** [comment]
i have updated the correlation file. You guys can have a look at it. Tanx

**Comment by KhairatTijani** [comment]
hello guys. The Research Question and Visualization assignment is live and due on Friday. Please let us all meet after data mining class in school today to finalize the analysis and submit today. Thanks x

**Reply by feekieboy** [reply]
Noted and will be there

**Comment by ao23aeq** [comment]
Good evening guys. I uploaded the RScript for data cleaning and updated the summary of the analysis.
Everyone should please "pull".

**Reply by feekieboy** [reply]
Done

**Comment by dpor16** [comment]
thanks @ao23aeq I'm having a little issue pulling from my machine. can anyone help me out pls? it just stopped pulling yesterday

**Comment by dpor16** [comment]
@feekieboy thanks for the guide, error has been fixed, pull and push working perfectly

**Reply by feekieboy** [reply]
You are welcome

**Comment by KhairatTijani** [comment]
Hello guys, I just updated my part of the analysis "descriptive and exploratory analysis" and also updated the Rscript. Please let us all meet up after Team Research Practical Session today at the Forum so we can submit our assignment. Thanks. @feekieboy over to you for correlation analysis.

**Reply by feekieboy** [reply]
ok

**Comment by dpor16** [comment]
Hi guys, I've just uploaded the cleaned dataset on git for record purpose, pls pull and have a look.

**Comment by ao23aeq** [comment]
Hi guys. Hope everyone is done with their roles and responsibilities. Thank you

**Comment by ao23aeq** [comment]
Thank you, team, for your hard work, collaboration, and dedication in completing the group assignment successfully!
